webapp/src/app.tsx

```diff
@@ -2914,6 +2914,18 @@ export class ProjectView
             });
     }
 
+    newProjectMaybeWithProjectCreationOptions(firstProject: boolean) {
```

Comment:
We'd absolutely welcome a better name for this method. An alternative could be just to duplicate this code in both `accessibility.tsx` and `projects.tsx` in their own `newProject` methods that then call `this.props.parent.newProject`. The latter approach does make it more likely that this bug will reoccur if a new entry point (button) to create a new project is introduced in the app.

Reply:
oh yeah, that one's a mouthful... perhaps `newUserCreatedProject`? would also be nice to convert this into async/await while you're here

Reply:
That sounds good. Both changes made here [aea20a3](https://github.com/microsoft/pxt/pull/10382/commits/aea20a3cefe29c01ac7bb7d82456cf07ad3fa184).